projekt_do_pracy_na_labach/fiszki_backend/Dockerfile

```diff
@@ -1,2 +1,40 @@
 FROM python:3.10
 
+WORKDIR /app
+
+COPY . /app
+
+RUN pip install -r requirements.txt
+
+ENV DATABASE_URL="mongodb://mongo:27017"
+
+EXPOSE 8000
+
+CMD ["python","main.py"]
+
+#Komendy:
+
+# 3.5:
+# docker build --tag fiszki_backend . 
+# docker run -p 8000:8000 --name fiszki_backend fiszki_backend
+
+# 4.0:
+# docker network create siec
+# docker build --tag fiszki_backend .
+# docker run -p 8000:8000 --network=siec --name fiszki_backend fiszki_backend 
+# docker run -p 27017:27017 --network=siec --name mongo mongo
+# docker network inspect siec
+# ta część dalej nie działa nawet po dodaniu linijki 7 (tak jak na labach)
```

Comment:
W czasie labów zapomniałem że prawdopodobnie działałeś cały czas na jednym okienku terminalu. (To nie jest złe).
Chodzi o to, że prawdopodobnie włączałeś kontener, a potem go wyłączałeś by móc znowu coś robić w terminalu. Spróbuj użyć komendy docker run z flagą `-d` i potem dać `docker network inspect siec`. 
Ogólnie u mnie działa więc zaliczam ale sobie przetestuj ;)  

Przykładowe wywołanie:
```
docker run -d -p 27017:27017 --network=siec --name mongo mongo
```